### PR TITLE
docs: add contribute new locale instruction

### PR DESCRIPTION
**Pull request by reslear**
### 📝 Checklist

- [x] All commits follow the Conventional Commit format
- [x] The PR's title follows the Conventional Commit format

### CODE REVIEW

**Comment by dargmuesli** [comment]
Closing as stale.

**Comment by reslear** [comment]
not stale

**Comment by dargmuesli** [comment]
When would you expect to answer my questions above?

**Comment by reslear** [comment]
I'm sorry, I don't see your questions.) 
Screenshot 2024-03-21 at 18 47 01

**Comment by dargmuesli** [comment]
Oh ok, I've resubmitted them. I think they should show now. Sorry for the confusion :pray:

File: README.md [hunk 1]
```diff
@@ -278,3 +278,11 @@ Every property that includes a `{ en: ... }` value is a translatable property th
...
 [npm-downloads-src]: https://badgen.net/npm/dm/@dargmuesli/nuxt-cookie-control
 [npm-downloads-href]: https://npmjs.com/package/@dargmuesli/nuxt-cookie-control
+
+## Contribute
+
+For new locales, adhere to the ISO 639-1 code, and incorporate alterations to the code similar to the pull request (PR) example provided at https://github.com/dargmuesli/nuxt-cookie-control/pull/194.
+
+## License
+
+[MIT](./LICENSE)
```
[the remark this answers is not in the document]
Comment (reslear) [reply README.md:288]: I like it better when everything is in a visible place, like for example for handy when npm is watching, but you are the boss :)

File: README.md [hunk 2]
```diff
@@ -278,3 +278,11 @@ Every property that includes a `{ en: ... }` value is a translatable property th
 
 [npm-downloads-src]: https://badgen.net/npm/dm/@dargmuesli/nuxt-cookie-control
 [npm-downloads-href]: https://npmjs.com/package/@dargmuesli/nuxt-cookie-control
+
+## Contribute
+
+For new locales, adhere to the ISO 639-1 code, and incorporate alterations to the code similar to the pull request (PR) example provided at https://github.com/dargmuesli/nuxt-cookie-control/pull/194.
```
[the remark this answers is not in the document]
Comment (reslear) [reply README.md:284]: moved to CONTRIBUTING.md

**Review by dargmuesli — APPROVED** [review]
Thank you for your suggestions! :green_heart:

**Pull request merged**